splitter.js

```diff
@@ -33,27 +34,52 @@ let isTestFilesOnSyncWithTestResults = function (testFiles, testResultFiles) {
   let missingTests = [];
   let testResultFilesMap = new Map();
   testResultFiles.forEach((testResultFile) => {
-    let fileName = path.parse(testResultFile).name.split(".").pop();
+    let xml = JSON.parse(convert.xml2json(fs.readFileSync(testResultFile)));
+    let fileNameData = xml.elements[0].attributes.name.split(".");
+    let fileName = fileNameData.pop();
+    let packageName = fileNameData.join(".");
     if (!testResultFilesMap.has(fileName)) {
-      testResultFilesMap.add(1, fileName);
+      testResultFilesMap.add([packageName], fileName);
     } else {
-      testResultFilesMap.add(testResultFilesMap.get(fileName) + 1, fileName);
+      let testResultFilePackages = testResultFilesMap.get(fileName);
+      testResultFilePackages.push(packageName);
+      testResultFilesMap.add(testResultFilePackages, fileName);
     }
+    testResultFilesMap.get(fileName);
```

Comment:
Can this be removed?

Reply:
Ah right, this was part of my test, will remove this.

Reply:
Removed